tests/test_pointset.py

```diff
@@ -121,28 +122,29 @@ def test_center_of_mass():
     assert np.allclose(pset.center_of_mass(), [0.5, 0.0, 0.5])
 
 
+@pytest.mark.filterwarnings("ignore:Points is not a float type")
 def test_points_to_double():
     np_points = np.array([[1, 2, 3]], np.int64)
     pset = pyvista.PointSet(np_points)
     assert pset.points_to_double().points.dtype == np.double
 
 
 def test_translate():
-    np_points = np.array([1, 2, 3], np.int64)
+    np_points = np.array([1, 2, 3], np.float32)
```

Comment:
Since we're testing both values of `force_float` separately, I figured we can just change all the other sources of the warning to use `float32` from the get go. The only thing missing then is a "check that default is `force_float=True`" test, but I'm not sure how useful that is.